osbs/conf.py

```diff
@@ -362,6 +363,15 @@ def get_build_imagestream(self):
     def get_proxy(self):
         return self._get_value("yum_proxy", self.conf_section, "yum_proxy")
 
+    def get_labels(self):
+        labels_list = self._get_value("labels", self.conf_section, "labels")
+        if labels_list is None:
+            return {}
+        # When read from config file, it needs to be deserialized
+        if type(labels_list) is not list:
+            labels_list = json.loads(labels_list)
```

Comment:
Yes. It does look odd, but I don't suppose anyone would use it in the config. We could enhance the Configuration class to control the sources of certain item, but as it stands it does not support this behavior. Is it worth adding it?